src/main/java/com/amazonaws/kinesisvideo/java/client/KinesisVideoJavaClientFactory.java

```diff
@@ -66,17 +70,56 @@ public static KinesisVideoClient createKinesisVideoClient(
             @Nonnull final Regions regions,
             @Nonnull final AWSCredentialsProvider awsCredentialsProvider)
             throws KinesisVideoException {
+
+        return createKinesisVideoClient(regions,
+                awsCredentialsProvider,
+                null, true, null);
+    }
+
+    @Nonnull
+    public static KinesisVideoClient createKinesisVideoClient(
+            @Nonnull final Regions regions,
+            @Nonnull final AWSCredentialsProvider awsCredentialsProvider,
+            @Nonnull final String endpointOverride,
+            @Nullable final IPVersionFilter ipVersionFilter)
+            throws KinesisVideoException {
+        return createKinesisVideoClient(regions, awsCredentialsProvider, endpointOverride, false, ipVersionFilter);
```

Comment:
Since `isLegacyEndpoint` is not applicable for when `endpointOverride` is used, could we consider having the `isLegacyEndpoint` Nullable and passed as null here?

Reply:
Sure we can do that.